content/security-humanops.rst

```diff
@@ -0,0 +1,231 @@
+Security
+########
+
+:menu_order: 009
+
+.. contents::
+
+
+Overview
+========
+
+The following is more of a public-facing overview or a process rather than focusing on the step-by-step portion of security. We also have additional security requirements - sysadmin will utilize more stringent security, travelers will want to utilize more stringent travel security processes, developers will have secure coding practices etc.
+
+The focus is on the human-side of security with some suggestions of tools and tactics.
+
+
+Onboarding
+==========
+
+When onboarding a team member, they will have limited access to Pro:Atria until such a time arrives when further credentials can be provided.
+
+There may instances when certain items are shared to the onboarding member's non-Pro:Atria address or accounts. Once a member is part of the team all use of services / apps related to ProAtria will need to be switched into ProAtria emails.
```

Comment:
For the last part, I added: _For example, if your personal email was added in the Google Analytics account during onboarding, please ensure that your personal email address is replaced by the new Pro:Atria email address._